plugin.go

```diff
@@ -93,6 +108,54 @@ func (p *Plugin) Exec() error {
 				return fmt.Errorf("Error: timed out waiting on a build for %s.\n", entry)
 			// Got a tick, we should check on the build status
 			case <-tick:
+				// first handle the deploy trigger
+				if len(p.Deploy) != 0 {
+					build, err := client.BuildLast(owner, name, branch)
```

Comment:
Why do you fetch the last build here? In case of the last successful build you don't need that.